tmpfiles.sh

```diff
@@ -97,14 +123,86 @@ splitpath() {
 
 _restorecon() {
     local path=$1
+	if ! owned_by_root $1 ; then
+		echo "cowardly refusing to restorecon" >&2
+		return 103
+	fi
     if [ -x /sbin/restorecon ]; then
         dryrun_or_real restorecon -F "$(splitpath "${path}")"
     fi
 }
 
+_chmod() {
+    local path=$2 mode=$1
+	if ! owned_by_root "${path}" ; then
```

Comment:
bad indentation in this code

Reply:
Changed in commit 4f4576a6781fb64a6c798d131f8309802c2f9c3a